.eslintrc.js

```diff
@@ -22,5 +22,11 @@ module.exports = {
         'no-undef': 'off',
       },
     },
+    {
```

Comment:
Please remove this ignore rule. This is not a solution to the problem at hand, but an illustration of the scope creep commented on earlier.